plugins/woocommerce/src/Admin/Features/OnboardingTasks/TaskLists.php

```diff
@@ -138,7 +166,7 @@ public static function init_default_lists() {
 					'Appearance',
 				),
 				'event_prefix' => 'tasklist_',
-				'visible'      => Features::is_enabled( 'tasklist-setup-experiment-2' ),
+				'visible'      => self::is_experiment_treatment( 'woocommerce_tasklist_setup_experiment_2' ),
```

Comment:
I wonder if we should add ` && ! self::is_experiment_treatment( 'woocommerce_tasklist_setup_experiment_1' )` to this one, just incase there is some experiment overlap and both of these experiments are turned on for a user. That way not both task lists are shown and it defaults to the first experiment?

Reply:
Seems like a safe way to handle this just in case segments are ever updated. 👍